src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -278,6 +282,22 @@ public WorkspaceViewModel(WorkspaceModel model, DynamoViewModel dynamoViewModel)
             Notes_CollectionChanged(null, new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Add, Model.Notes));
             foreach (var c in Model.Connectors)
                 Connectors_ConnectorAdded(c);
+
+            if (Model is HomeWorkspaceModel)
+            {
```

Comment:
This is so dangerous, we should have a `HomeWorkspaceViewModel` and `CustomNodeWorkspaceViewModel`.